admin/class-meta-columns.php

```diff
@@ -51,6 +62,20 @@ public function setup_hooks() {
 		}
 
 		add_filter( 'request', [ $this, 'column_sort_orderby' ] );
+
+		// Hook into tablenav to get the indexables.
```

Comment:
I'm missing the why here. Could you elaborate a bit here?
Something like: retrieving the indexables in one query beforehand, because that is more efficient than one by one later on.